client/src/main.jsx

```diff
@@ -3,11 +3,16 @@ import ReactDOM from "react-dom/client";
 
 import { createBrowserRouter, RouterProvider } from "react-router-dom";
 
+import Homepage from "./pages/Home";
 import App from "./App";
 
 const router = createBrowserRouter([
   {
     path: "/",
+    element: <Homepage />,
```

Comment:
Since we have the page "Home" you should consider to rename this component with something more suitable : "welcoming" or "reception" for example